### DISCUSSION TITLE
Kontrol S4 MK1 Driver MacBook m2

### DISCUSSION THREAD

**Discussion by Djandersound1** [Opa-/x1-mk1-usb2midi#8] [Ideas]
Aqui está a tradução do seu texto para o inglês:

"I bought the Kontrol S4 MK1 and saw that the driver is not compatible with the MacBook M2. Can you make the driver work? There are several people online looking for this; I believe you could make some money. I could also negotiate with you .

You can send me an email or a WhatsApp message.

ander.3g@gmail.com

WhatsApp: +55 67 99250-1871"

**Comment by Opa-** [comment]
Converting S4 Mk1 USB events to MIDI was done for Linux https://github.com/blaxpot/traktor-s4-mk1-midify but it rely on a Python library (https://python-evdev.readthedocs.io/en/latest/) that is not compatible with macOS so it should be rewritten somehow.

Unfortunately I do not have a S4 Mk1 myself to begin with, can't do much without it.

**Reply by Djandersound1** [reply]
"Is it possible to do this remotely? What apps would we need to install? Is there any way to emulate the Mac?

I would really like to be able to use my Kontrol S4 MK1."

**Reply by Opa-** [reply]
Sorry I can't work on that remotely
